### PR TITLE
webpack, spec, and npm cleanups/updates

### PR DESCRIPTION
**Pull request by martinpitt**
Catching up with starter-kit, minimize webpack.config.js/Makefile difference.

### CODE REVIEW

**Comment by martinpitt** [comment]
I visually tested this on my laptop's OSTree. No visual difference to master, works fine.

**Comment by martinpitt** [comment]
@marusak , @KKoukiou : Can I bother you with this today? It looks a bit scary first, but it's really just backports of starter-kit canges, and each commit by itself is fairly small. I'd like to backport the more recent translation cleanups as well.

**Comment by marusak** [comment]
packit is sad:
```
The create-archive action did not output a path to the generated archive. Please make sure that you have valid path in the single line of the output.
```
Seems easy to fix. Not sure yet if related to these changes (gonna take a look at it now), just wanted to point it out :)

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,7 @@ dist/
 package-lock.json
```
Comment (marusak) [inline .gitignore:9]: The sha in this commit message (Copy patternfly import files from cockpit project automatically ) is incorrect. The correct one is https://github.com/cockpit-project/starter-kit/commit/203604a463f900cfef0980a629ddfe9fb14c39ad

**Review by marusak — DISMISSED** [review]
Just one commit message is incorrect. Feel free to land though :)

File: .gitignore (hunk 1 above)
Comment (martinpitt) [reply .gitignore:9]: Ugh, thanks for spotting! Fixed, can you please re-approve?

**Comment by martinpitt** [comment]
@marusak : Packit has been broken for a long time indeed. It's on my January list to fix this.

**Review by marusak — APPROVED** [review]
Thanks!

**Pull request merged**